<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clientDialog</class>
 <widget class="QDialog" name="clientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="detailsButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>105</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Détails</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>350</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="reserveButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>105</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Réserver</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Aucun</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12-25</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Étudiant</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sénior</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mort vivant</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Nombre :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Abonnement :</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>105</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
